includes/ManageWikiHooks.php

```diff
@@ -1,5 +1,12 @@
 <?php
 class ManageWikiHooks {
+	public static function fnManageWikiSchemaUpdates( DatabaseUpdater $updater ) {
```

Comment:
Sounds better if it is onManageWikiSchemaUpdates.

Reply:
meh, I pretty much just copied this over from CreateWiki

Reply:
ah ok